src/core/model/table/relational_schema.h

```diff
@@ -30,6 +30,7 @@ class RelationalSchema {
     std::unique_ptr<Vertical> empty_vertical_;
 
     RelationalSchema(std::string name);
+    RelationalSchema(RelationalSchema&& other);
```

Comment:
It's enough to use default move constructor. And let's define other constructors/operators from rule of 5 (destructor is already defined):
```suggestion
    RelationalSchema(RelationalSchema const& other) = delete;
    RelationalSchema& operator=(RelationalSchema const& rhs) = delete;
    RelationalSchema(RelationalSchema&& other) noexcept = default;
    RelationalSchema& operator=(RelationalSchema&& rhs) noexcept = default;
```